lambda/src/main/resources/db/migration/V21__change_consignment_metata_foreign_key.sql

```diff
@@ -0,0 +1,13 @@
+ALTER TABLE "ConsignmentMetadata" DROP CONSTRAINT "ConMetadata_Property_fkey";
+ALTER TABLE "ConsignmentMetadata" ALTER COLUMN "PropertyId" TYPE text;
+ALTER TABLE "ConsignmentMetadata" RENAME COLUMN "PropertyId" TO "PropertyName";
+ALTER TABLE "ConsignmentMetadata" ADD CONSTRAINT
+    "ConMetadata_PropertyName_fkey" FOREIGN KEY ("PropertyName") REFERENCES "ConsignmentProperty" ("Name")
+        MATCH SIMPLE
+        ON UPDATE NO ACTION
+        ON DELETE NO ACTION
+        NOT VALID;
```

Comment:
Is `NOT VALID` necessary? It sounds like it's only useful if there's a massive amount of data in the table, which isn't true here. I'd rather not set a precedent of using it, in case it causes inconsistent data in the future.

Reply:
Removed